### ISSUE TITLE
损失函数的使用方法

### ISSUE THREAD

**Issue by qiangruoyu** [idstcv/DR_loss#3]
希望作者能详细写一下损失函数的备注，例如输入代表的是什么，shape是多少，这样更方便判断在其他情况下不能不能使用，感谢。

**Comment by qian-qi** [comment]
因为我们的loss现在嵌套在maskrcnn_benchmark中，所以用的是它的输入格式. 实际中只需要能输入30行和31行的positive和negative anchor的logits就行了. 可以对照着我们的paper看，paper中写的比较详细.

**Comment by qiangruoyu** [comment]
> 因为我们的loss现在嵌套在maskrcnn_benchmark中，所以用的是它的输入格式. 实际中只需要能输入30行和31行的positive和negative anchor的logits就行了. 可以对照着我们的paper看，paper中写的比较详细.

logits是什么呢？shape是（batch_size,class,w,h）吗？

**Comment by qian-qi** [comment]
logits包含了所有anchor的logits，它的总数应该还要乘上每个location的anchor的数量. 具体可以看一下RetinaNet的描述(Fig. 3 in https://arxiv.org/pdf/1708.02002.pdf).

**Comment by qiangruoyu** [comment]
> logits包含了所有anchor的logits，它的总数应该还要乘上每个location的anchor的数量. 具体可以看一下RetinaNet的描述(Fig. 3 in https://arxiv.org/pdf/1708.02002.pdf).

对着maskrcnn刚刚改好，损失函数用起来挺简单，但是论文里的证明着实不少，可见工作还是蛮多的。希望能超越focal loss。万分感谢！

**Comment by mytk2012** [comment]
你好，问下logits是指的损失函数用的logistics这个吗，所有anchor的logits是指的什么，shape是（batch_size,class,w,h）吗？

**Comment by long2double** [comment]
logits和targets的shape分别是什么？
